staging/src/k8s.io/legacy-cloud-providers/gce/gce_loadbalancer_internal.go

```diff
@@ -982,12 +982,18 @@ func forwardingRulesEqual(old, new *compute.ForwardingRule) bool {
 	if err != nil {
 		klog.Errorf("forwardingRulesEqual(): failed to parse resource URL from new FR, err - %v", err)
 	}
-	return (old.IPAddress == "" || new.IPAddress == "" || old.IPAddress == new.IPAddress) &&
+	resourcEqual := false
```

Comment:
Nit: resourceEqual.

Actually, since we are specifically checking only the resourceID, we can name it "resourceIDEqual"

Reply:
Updated